index.html

```diff
@@ -14,17 +14,21 @@
     <main>
         <div class="intro">
             <header>
-              <div class="nav_conatiner">
+              <div class="nav_container">
                 <nav class="nav-bar">
-                  <a href="#"><h1 class="logo">My Logo</h1></a>
+                  <a href="#"><h3 class="logo">My Logo</h3></a>
+                  <div class="humburger">
+                    <button id="open-menu-btn"><i class="uil uil-bars"></i></button>
+                    <button id="close-menu-btn"><i class="uil uil-multiply"></i></button>
+                  </div>
                   <ul class="nav-links">
-                    <li><a class="item" href="#">Portfolio</a></li>
-                    <li><a class="item" href="#">About</a></li>
+                    <li><a class="item" href="#profile">Portfolio</a></li>
+                    <li><a class="item" href="#about">About</a></li>
                     <li class="contact">
-                      <a class="item" href="#">Contact</a>
+                      <a class="item" href="#contact">Contact</a>
```

Comment:
- [ ] Kindly when I open the menu and close it, when I try to open it again it does not show the blue background, it shows only the text over the original content. Kindly fix this too.

![image](https://user-images.githubusercontent.com/35535130/173465872-541f93d7-27a8-4263-ad99-68907ff65e02.png)